skyportal/handlers/api/source.py

```diff
@@ -740,6 +731,14 @@ def get_sources(
                 "Invalid commentsFilterAuthor value -- must provide at least one string value"
             )
     if localization_dateobs is not None:
+
+        # This grabs just the IDs so the more expensive localization in-out
+        # check is done on only this subset
+        obj_ids = session.scalars(obj_query).all()
```

Comment:
What happens if the filters above don't filter anything and you get back *all* the obj ids?

Reply:
I will put a cap and raise an error.